package-lock.json

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
Please remove this lockfile. We use yarn and lockfiles are generated and updated by me.